Hangman/Hangman.py

```diff
@@ -0,0 +1,64 @@
+from Board import Board
+from Generate_word import GenerateWord
+from User_Player import UserPlayer
+
+
+def is_alphabet(letter) -> bool:
+    if len(letter) > 1:
+        return False
+    return letter in "abcdefghijklmnopqrstuvwxyz"
+
+
+class Hangman:
+    def __init__(self):
+        self.board = Board()
+        self.userPlayer = UserPlayer()
+
+    def ask_for_game(self) -> bool:
+        want_game = self.userPlayer.want_game()
+        return want_game
+
+    def start_game(self) -> None:
+        self.board.set_word(GenerateWord.get_random_word())
+        self.run_game()
+
+    def run_game(self) -> None:
+        if self.board.has_chances():
+            self.guessing()
+            if self.board.guessed_it():
+                print("you have guessed the word ", end="\n\n")
+                return
+            else:
+                self.run_game()
+        else:
+            print("no chances left")
+            print("word is ", self.board.get_word(), end="\n\n")
+            return
+
+    def guessing(self) -> None:
+        print("You have {} chance ".format(self.board.get_chance()))
+        letter = self.userPlayer.guess_letter()
+
+        if not is_alphabet(letter):
+            print("It is not  an alphabet try an alphabet")
+        elif self.board.word_has(letter):
```

Comment:
what if i have already guessed that letter ? that would just say right guess! but it should instead say already guessed , choose a different letter